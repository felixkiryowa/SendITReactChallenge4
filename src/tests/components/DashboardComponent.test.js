import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import DashboardComponent from '../../components/Parcels/index';

describe('Sample Articles Component', () => {
    let mountApp;
    const props = {
        handleSubmit: jest.fn(),
        handleFromPlace: jest.fn(),
        handleFromPlace: jest.fn(),
        handleWeightChange: jest.fn(),
        errors: {

        },
    };
    beforeEach(() => {
        mountApp = mount(<DashboardComponent {...props} />);
    });

    it('Should render without crashing', () => {
        expect(mountApp).toMatchSnapshot();
    });
});
