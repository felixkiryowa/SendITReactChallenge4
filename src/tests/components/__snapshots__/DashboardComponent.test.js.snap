// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample Articles Component Should render without crashing 1`] = `ReactWrapper {}`;
